packages/editor/ProseMirror/createEditor.ts

```
@@ -147,6 +160,48 @@ const createEditor = async (
+			const isNodeSlice = slice.openStart === 0 && slice.openEnd === 0 && slice.content.childCount === 1;
+			if (isNodeSlice) {
+				tr.replaceRangeWith(insertPos, insertPos, slice.content.firstChild);
+			} else {
+				tr.replaceRange(insertPos, insertPos, slice);
+			}
+
+			if (!tr.doc.eq(beforeInsert)) {
+				const mappedPos = tr.mapping.map(insertPos, 1);
+				const clampedPos = Math.min(mappedPos, tr.doc.content.size);
+				tr.setSelection(Selection.near(tr.doc.resolve(clampedPos), -1));
+				view.focus();
```

ESLint

[error] 'focus' is restricted from being used. Please use focusHandler::focus() instead

(no-restricted-properties)